styles.css

```diff
@@ -72,6 +72,15 @@
     text-decoration: none;
 }
 
+.internal-button {
```

Comment:
Understand why this has a generic name - what does `internal` mean - should it share any settings with other buttons?